app/dcr_monitor/tee_backend/confidential_space.go

```diff
@@ -208,7 +210,54 @@ func (c *TEEProviderGCPConfidentialSpace) getConfidentialSpaceInsertInstanceRequ
 	return req
 }
 
+func (c *TEEProviderGCPConfidentialSpace) createWorkloadIdentityPoolProvider(name string, digest string) error {
```

Comment:
workload identity pool is used for allowing the confidential space instance to use the kms key. Since the kms key has been totally removed, we should think whether the workload identity is required any more.